style.css

```diff
@@ -0,0 +1,250 @@
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  padding-top: 10px;
+  padding-left: 12px;
+}
+
+.title1 {
+  font-family: Poppins, Geneva, Tahoma, sans-serif;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 52px;
+  padding-top: 50px;
+  color: #172b4d;
+}
+
+.description {
+  padding-right: 10px;
+  font-family: Poppins, Geneva, Tahoma, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+}
+
+.connect {
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
+}
+
+a:hover { color: #7f8cff; }
+
+.social {
+  display: flex;
+  width: 300px;
+  justify-content: flex-start;
+}
+
+.social ul {
+  list-style: none;
+  padding-left: 0%;
+}
+
+.social li {
+  float: left;
+  margin-inline-start: 15px;
+}
+
+.works-container {
+  display: grid;
+  width: 100%;
+  margin: 0 auto;
+  padding-top: 50px;
+  grid-template-columns: auto;
+}
+
+.snapshot {
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+  width: 95%;
+  height: auto;
+}
+
+.flex-card {
+  width: 100%;
+  flex-direction: row;
+  border-radius: 25px;
+  padding-bottom: 20px;
+  padding-top: 50px;
+}
+
+.flex-card h1 {
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  padding-left: 7px;
+}
+
+.flex-card p {
+  margin-left: 10px;
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.flex-card ul {
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding-left: 5px;
+}
+
+.element1 {
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  padding-right: 10px;
+}
+
+.element2 {
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: 600;
+  font-size: 17px;
+  color: #7a869a;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.language-buttons {
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  color: #6070ff;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+  background-color: #ebebff;
+  border: none;
+  padding: 8px;
+  margin: 4px 2px;
+  border-radius: 12px;
+}
+
+.bottom-list {
+  list-style: none;
+  padding-left: 0%;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.p-button {
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  color: #6070ff;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  background-color: transparent;
+  border: 2px solid #838de9;
+  padding: 15px;
+  margin: 4px 2px;
+  border-radius: 12px;
+}
+
+.p-button:hover { background-color: #7f8cff; }
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  padding-left: 10px;
+  padding-top: 10px;
+  padding-bottom: 25px;
+  box-shadow: 2px 2px 2px #888;
+}
```

Comment:
- [ ] Kindly, adjust "About Myself "section to have the the margins (top, bottom, left, right) defined in the design. 

Your project Look like this : 
![Captur12](https://user-images.githubusercontent.com/93598189/186737991-8326ae8b-3faf-4712-b572-add8248195a0.PNG)

The Figma design Look likes this : 
![1234](https://user-images.githubusercontent.com/93598189/186738057-ba06d47d-5801-4e2a-9fde-e88418653bda.PNG)

Reply:
Ok, problem solved